### PR TITLE
feat: Update Search Console details page headings

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue #460.

Adds the new headings requested in #460 and removes unneeded props (eg. `header` and `title` from existing `<Layout />` components.

### Before

Screenshot 2019-09-04 16 38 18

### After

Screenshot 2019-09-04 16 34 19

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/analytics/dashboard-details/dashboard-details-widget-top-acquisition-sources.js [hunk 1]
```diff
@@ -17,14 +17,16 @@
  */
 
 /**
- * Internal dependencies
+ * External dependencies
```
Comment (felixarntz) [inline assets/js/modules/analytics/dashboard-details/dashboard-details-widget-top-acquisition-sources.js]: Let's remove this comment as there are no external dependencies here.

File: assets/js/modules/search-console/dashboard-details/dashboard-details-widget-keyword-table.js [hunk 2]
```diff
@@ -17,38 +17,47 @@
  */
 
 /**
- * Internal dependencies
+ * External dependencies
```
Comment (felixarntz) [inline assets/js/modules/search-console/dashboard-details/dashboard-details-widget-keyword-table.js]: See above.

File: assets/js/modules/search-console/dashboard-details/dashboard-details-widget-search-funnel.js [hunk 3]
```diff
@@ -17,40 +17,54 @@
  */
 
 /**
- * Internal dependencies
+ * External dependencies
```
Comment (felixarntz) [inline assets/js/modules/search-console/dashboard-details/dashboard-details-widget-search-funnel.js]: See above.

**Comment by felixarntz** [comment]
@adamsilverstein Just wondering, we don't have visual regression tests for this screen? I was surprised to find that none failed.

File: assets/js/modules/analytics/dashboard-details/dashboard-details-widget-top-acquisition-sources.js (hunk 1 above)
Comment (tofumatt) [reply assets/js/modules/analytics/dashboard-details/dashboard-details-widget-top-acquisition-sources.js]: I'm confused; why are these components considered "external"? They are internal to the project and only arranged to "look like" non-project dependencies via the webpack config. What's the thinking behind labelling them external when they're inside the project's source code?

**Comment by adamsilverstein** [comment]
> @adamsilverstein Just wondering, we don't have visual regression tests for this screen? I was surprised to find that none failed.

I'm surprised as well @felixarntz - probably because these tests were designed to test and catch regressions in individual components rather than complete plugin screens - which is clear from the reference images in `tests/backstop/reference`. Since the components used for the detail page are the same as on the dashboard, they are already tested there. It might be worth adding some additional tests for entire UI pages to catch these types of changes.

File: assets/js/modules/analytics/dashboard-details/dashboard-details-widget-top-acquisition-sources.js (hunk 1 above)
Comment (adamsilverstein) [reply assets/js/modules/analytics/dashboard-details/dashboard-details-widget-top-acquisition-sources.js]: Historical context: originally the distinction was some components were part of the module (eg analytics here) and some were "common" modules provided by the core plugin (in the original implementation, modules were structurally more distinct from the core plugin). These were called "external" (to the module) dependencies, and imported using the `GoogleComponents` or `GoogleUtils` prefix (which is an alias in webpack).

**Comment by tofumatt** [comment]
It sounds like what you're looking for here (and what might make more sense for full-page layouts) are [Jest Snapshots](https://jestjs.io/docs/en/snapshot-testing) over visual regression tests. The snapshots are a _lot_ faster to run and update than the visual regression tests, which require standing up another server and a fair bit of overhead.

Ensuring the page layout is consistent is good, but we should look toward lighter-weight tests for future tests—the existing E2E and visual regression tests both take a considerable amount of time to run (9+ minutes for E2E and `5m 55s` for visual regression tests on my 2017 MacBook Pro). Especially because those tests don't run "partially" based on diffed files (as Jest will), we should probably refrain from using them for large pages that are often likely to change—if we snapshop test those pages instead we can iterate on issues faster 😄

**Pull request merged**